shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/metadata/rule/ClassTest1.java

```diff
@@ -0,0 +1,4 @@
+package org.apache.shardingsphere.infra.metadata.rule;
+
+public final class ClassTest1 extends ShardingSphereRuleFake {
```

Comment:
It is better to rename `Fake` as `Fixture`, just keep consist with original codes

Reply:
Please avoid name class name as `ClassTest1`, it likes a temporary name.